zerver/webhooks/netlify/tests.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+
+from zerver.lib.test_classes import WebhookTestCase
+from zerver.models import get_system_bot
+
+class NetlifyHookTests(WebhookTestCase):
+    STREAM_NAME = 'netlify'
+    URL_TEMPLATE = u"/api/v1/external/netlify?stream={stream}&api_key={api_key}"
+
+    def test_building_message(self) -> None:
+        expected_subject = u"objective-jepsen-35fbb2"
+        expected_message = (
+            u'State: building\n'
+            u'Name: objective-jepsen-35fbb2\n'
+            u'Branch: master'
+        )
+
+        self.send_and_test_stream_message('building', expected_subject, expected_message,
+                                          content_type="application/json")
```

Comment:
There is also an HTTP header value that accompanies each fixture that indicates the event type of the payload. I would recommend incorporating that into your implementation as well! :)

See [how our GitHub webhook handles HTTP headers of this sort](https://github.com/zulip/zulip/blob/master/zerver/webhooks/github/view.py#L445) for further information! :)